src/frontend/src/lib/i18n/en.json

```diff
@@ -636,11 +636,11 @@
 		},
 		"what": {
 			"text": {
-				"label": "$oisy_short's features",
-				"title": "Utility",
-				"hold_crypto": "<strong>Hold crypto across networks</strong><br>Manage and transact with native Ethereum, Bitcoin <i>(coming soon)</i>, and Internet Computer assets. Powered by <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/chainfusion\">Chain Fusion</a>.",
-				"use_eth_dapps": "<strong>Use with Ethereum dapps</strong><br>$oisy_name is fully compatible with Ethereum Defi like <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://uniswap.org/\">Uniswap</a> thanks to the <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://walletconnect.com/\">WalletConnect</a> framework.",
-				"use_ic_dapps": "<strong>Use with Internet Computer dapps (Coming soon)</strong><br>Use your $oisy_name wallet in any <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/ecosystem\">dapp on the Internet Computer</a> that supports the signer standards, including <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://oc.app/\">OpenChat</a>, <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://www.icpswap.com/\">ICPSwap</a>, and the <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://nns.ic0.app/\">NNS dapp</a>."
+				"label": "Wallet Features",
+				"title": "$oisy_short Wallet Features",
+				"hold_crypto": "<strong>Hold crypto across networks</strong><br>Manage and transact with native Ethereum, Bitcoin, and Internet Computer assets. Powered by <a rel=\"noreferrer noopener\" target=\"_blank\" href=\"https://internetcomputer.org/chainfusion\">chain fusion</a>.",
```

Comment:
Shouldn't Chain Fusion be upper case on the first letter? I remember this was discussed in the past, but I am not sure what is the official denomination

Reply:
TBH I matched figma because that's in the ticket. I don't know about discussions.
<img width="663" alt="image" src="https://github.com/user-attachments/assets/d5f49ffc-4855-42c8-9132-63a66ee1db1e">

Reply:
maybe double-check with the design team, but I think it should be Chain Fusion because on the website is like that

https://internetcomputer.org/docs/current/developer-docs/multi-chain/overview

Reply:
Thanks, done